dist/documentation/themes/dark.js

```diff
@@ -0,0 +1,627 @@
+/**
+ * Do not edit directly
+ * Generated on Wed, 25 Jan 2023 17:56:51 GMT
+ */
+
+module.exports = {text : {emphasis:{
```

Comment:
this format resembles the file we were pulling from originally for the tiles - it just one object deeper to expose the `name`, `hex`, and `description`

since we have dark and light tokens now, i was thinking that the table could be under some `Tabs`, where you can flip between the light and dark colours